pages/dkp/kaptain/2.1.0/tutorials/ngc/index.md

```diff
@@ -0,0 +1,112 @@
+---
+layout: layout.pug
+navigationTitle: NGC catalog
+title: NVIDIA NGC catalog
+menuWeight: 30
+excerpt: Use NVIDIA's NGC catalog with Kaptain
+beta: false
+enterprise: false
+---
+
+Kaptain supports the usage of Nvidia's NGC catalog in networked environments, which you can access and set up via the Kubeflow UI. The NGC catalog includes containers, charts, pre-trained models, toolkits and more, which are optimized for GPU deployment and can now be used to run AI workflows in Kaptain. You can find all the available resources for your AI/ML workflow in the [NVIDIA NGC catalog][NGC_catalog].
+
```

Comment:
Add a note here in the lines of: 

Not all NGC available images have been tested, so support is limited. 
install SDK or other (see slides from presi) for added funtionality.